src/coreclr/vm/loaderallocator.cpp

```diff
@@ -2217,6 +2252,140 @@ PTR_OnStackReplacementManager LoaderAllocator::GetOnStackReplacementManager()
 #endif //
 #endif // FEATURE_ON_STACK_REPLACEMENT
 
+#ifndef DACCESS_COMPILE
+void LoaderAllocator::AllocateBytesForStaticVariables(DynamicStaticsInfo* pStaticsInfo, uint32_t cbMem)
+{
+    CONTRACTL
+    {
+        THROWS;
+        GC_TRIGGERS;
+        INJECT_FAULT(COMPlusThrowOM());
+    }
+    CONTRACTL_END;
+
+    if (cbMem > 0)
```

Comment:
Thoughts on removing this indentation and returning immediately? Unclear why we would ever request zero though.

Reply:
Eh, it's just a habit. My first job was writing C, and with the rule of no early returns and discouraged gotos, so I sometimes fallback to that.  (That and my monitor is really wide, so deep indenting has never bothered me until it gets to something like 48 characters deep.) I'll make it an early return as it is not a problem to do so.

We pass 0 so that we don't have to check elsewhere for 0, and can just unconditionally call this method. I used it to check to see how often we had a 0 size vs something else. This structure made it a bit easier to write that temporary test logic. Turned out the result of that analysis was pointless, but the structure remained.